modules/serverless-configuring-kube-rbac-proxy-resources-for-serving.adoc

```diff
@@ -0,0 +1,37 @@
+// Module included in the following assemblies:
+//
+// * knative-serving/kube-rbac-proxy-serving.adoc
+
+:_content-type: REFERENCE
+[id="serverless-configuring-kube-rbac-proxy-resources-for-serving_{context}"]
+= Configuring kube-rbac-proxy resources for Serving
+
+You can globally override resource allocation for the `kube-rbac-proxy` container by using the {ServerlessOperatorName} CR.
+
+[NOTE]
+----
+You can also override resource allocation for a specific deployment.
+----
+
+The following configuration sets Knative Serving `kube-rbac-proxy` minimum and maximum CPU and memory allocation:
+
+.KnativeServing CR example
+[source,yaml]
+----
+apiVersion: serving.knative.dev/v1beta1
```

Comment:
On a second look, this should be `apiVersion: operator.knative.dev/v1beta1`

Reply:
@rh-max check https://github.com/openshift-knative/serverless-operator/blob/main/test/v1beta1/resources/operator.knative.dev_v1beta1_knativeserving_cr.yaml#L1